### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by BlueDasher334** [Leather128/LeathersFunkinEngine#75] [Ideas]
Add a scroll speed editor in a mid of a song

add a event function like psych engine

Add a customisable note color option

Add a song maker

**Comment by Leather128** [comment]
scroll speed changes mid song would be part of the event function thing (which has already been planned since it's such a nice feature), wondering what you mean by 'song maker' (because we already have a charter), and i don't think i really can add a customizeable note color option considering that mods can make custom ui skins, which may cause the note colors to not be what you want.
